### PR TITLE
StateMachine based multi-client workflows

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: packages/frontend/.ember-cli.js [hunk 1]
```diff
@@ -12,5 +12,5 @@ module.exports = {
   disableAnalytics: false,
   liveReload: true,
   host: '0.0.0.0',
-  usePods: true,
+  componentStructure: 'nested',
```
Comment (knownasilya) [inline packages/frontend/.ember-cli.js]: @NullVoxPopuli what's this?

Comment (NullVoxPopuli) [reply packages/frontend/.ember-cli.js]: defaults the component blueprint generator to do:

`app/components/{component-name}/index.{js,hbs}`

Comment (knownasilya) [reply packages/frontend/.ember-cli.js]: Instead of `app/components/{component-name}.{js,hbs}`?

Comment (NullVoxPopuli) [reply packages/frontend/.ember-cli.js]: yup!

**Comment by NullVoxPopuli** [comment]
Superseded by: https://github.com/NullVoxPopuli/emberclear/pull/434

**Pull request closed without merging**